build.sbt

```diff
@@ -53,7 +53,7 @@ val sharedSettings = Seq(
     "org.scalacheck" %% "scalacheck" % "1.15.2" % "test",
     "org.scalatest" %% "scalatest" % "3.2.15" % "test",
     "org.scalatestplus" %% "scalatestplus-scalacheck" % "3.1.0.0-RC2" % "test",
-    "com.esotericsoftware" % "kryo-shaded" % kryoVersion
+    "com.esotericsoftware" % "kryo" % kryoVersion
```

Comment:
According to the description in the Kryo repository, it seems that `com.esotericsoftware.kryo:kryo5:5.5.0` should be used?

https://github.com/EsotericSoftware/kryo

![image](https://github.com/twitter/chill/assets/1475305/67f8e822-56c8-40ff-856f-a00d67e6d20b)

Reply:
Hi @LuciferYang,

I agree with you! Yes, it would be better to use com.esotericsoftware.kryo:kryo5:5.5.0. I have switched to versionless kryo5 package to fix all mimaReport issues. We can switch back to this artifact any time before it would be merged. (I have mentioned this in the PR's description as well). Related commit: https://github.com/twitter/chill/pull/747/commits/0a6dda1dba9e682ba010b8344939fe7f7d4af6b6

Currently the main issue here that we cannot go further with this PR because seems like there is no active maintainter of this twitter/chill repository. :-(  I am just a simple contributor who opened this PR.